skimage/morphology/__init__.py

```diff
@@ -1,3 +1,12 @@
+"""Collection of operators to carryout nonlinear operations
+related to the form or shape of features in images. The key
+operators are dilation and erosion.
+
+This operations are particularly suited for binary images
+(where pixels are represented as 0 or 1), although it can
+be extended to images of all types.
```

Comment:
This is a good start! :) Though could you try to more closely match the [numpydoc style](https://numpydoc.readthedocs.io/en/latest/format.html#short-summary)? E.g. a docstring should start with a one-line summary, etc.

Reply:
Thanks for pointing that out. I will keep in mind.